### PR TITLE
tikv: use 96MB as the default region size

### PR DESCRIPTION
**Pull request by siddontang**
PTAL @BusyJay @overvenus @javaforfun

### CODE REVIEW

**Comment by javaforfun** [comment]
LGTM

File: etc/config-template.toml [hunk 1]
```diff
@@ -108,10 +108,10 @@
 # A threshold to gc stale raft log, must >= 1.
 # raft-log-gc-threshold = 50
 # When entry count exceed this value, gc will be forced trigger.
-# raft-log-gc-count-limit = 196608
+# raft-log-gc-count-limit = 48000
 # When the approximate size of raft log entries exceed this value, gc will be forced trigger.
 # It's recommanded to set it to 3/4 of region-split-size.
-# raft-log-gc-size-limit = "192MB"
+# raft-log-gc-size-limit = "48MB"
```
Comment (BusyJay) [inline etc/config-template.toml]: I think the default value is 72MB.

Comment (siddontang) [reply etc/config-template.toml]: Follow https://github.com/pingcap/tikv/pull/2017/files#diff-d4c79fcc9faffa48ae170c90d8c0683aL96

Comment (BusyJay) [reply etc/config-template.toml]: But it's conflict with https://github.com/pingcap/tikv/pull/2313/files#diff-c9a441bf6785be28f64f5b07d40725a2R118

File: etc/config-template.toml [hunk 2]
```diff
@@ -108,10 +108,10 @@
 # A threshold to gc stale raft log, must >= 1.
 # raft-log-gc-threshold = 50
 # When entry count exceed this value, gc will be forced trigger.
-# raft-log-gc-count-limit = 196608
+# raft-log-gc-count-limit = 48000
```
Comment (BusyJay) [inline etc/config-template.toml]: This should be updated too.

**Comment by siddontang** [comment]
PTAL @BusyJay

**Pull request merged**